packages/twenty-front/src/modules/apollo/utils/encodeCursor.ts

```diff
@@ -0,0 +1,18 @@
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { isDefined } from '~/utils/isDefined';
+
+export const encodeCursor = (record: ObjectRecord) => {
+  if (!('id' in record) || !isDefined(record.id)) {
+    throw new Error('Record does not have an id');
+  }
+
+  const payload: {
```

Comment:
Not sure if we need to instantiate this now that you don't modify it?

Reply:
I don't want to put all records values inside the payload, it's why I'm instantiating a new object